content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -271,61 +315,73 @@ When you checkout the metadata on your newly created NFT, you should now see a
 The last thing you need to do is verify the NFT. This effectively just flips the
 `verified` field above to true, but it's incredibly important. This is what lets
 consuming programs and apps know that your NFT is in fact part of the
-collection. You can do this using the `verifyCollection` function:
+collection. You can do this using the `verifyCollectionV1` function:
 
 ```typescript
-await metaplex.nfts().verifyCollection({
-  mintAddress: nft.address,
-  collectionMintAddress: collectionNft.address,
-  isSizedCollection: true,
-});
+const metadata = findMetadataPda(umi, { mint: mint.publicKey });
+
+await verifyCollectionV1(umi, {
+  metadata,
+  collectionMint,
+  authority: umi.identity,
+}).sendAndConfirm(umi);
 ```
 
 ### Lab
 
-In this lab, we'll go through the steps to create an NFT using the Metaplex SDK,
-update the NFT's metadata after the fact, and then associate the NFT with a
-collection. By the end, you will have a basic understanding of how to use the
-Metaplex SDK to interact with NFTs on Solana.
+In this lab, we'll go through the steps to create an NFT using the Metaplex UMI
+framework, update the NFT's metadata after the fact, and then associate the NFT
+with a collection. By the end, you will have a basic understanding of how to use
+the Metaplex UMI and the mplTokenMetadata library to interact with NFTs on
+Solana.
 
 #### Part 1: Creating an NFT collection
 
 To begin, make a new folder and install the relevant dependencies:
 
-```
-npm i "@solana/web3.js" "@solana-developers/helpers "@metaplex-foundation/js"
+```bash
+npm i @solana/web3.js \
+npm i @solana/web3.js \
+npm i @solana-developers/helpers \
+npm i @metaplex-foundation/mpl-token-metadata \
+npm i @metaplex-foundation/umi-bundle-defaults \
+npm i @metaplex-foundation/umi-uploader-irys \
+&& npm i -save-dev esrun
```

Comment:
Just do this like

```npm i 1 /
  2 /
  3 / 
  4 
```

To be consistent with our other content.

Also think it's `--save-dev`